internal/test/envtest/environment.go

```diff
@@ -72,11 +71,10 @@ import (
 )
 
 func init() {
-	klog.InitFlags(nil)
-	logger := klogr.New()
+	logger := klog.Background()
```

Comment:
Wouldn't the replacement be `textlogger.NewLogger` ?

Reply:
That is what the godoc is saying, yup. I though it's better to use the same in envtest that we also use in main.go

Reply:
It falls back to some logger so it is not the same as in main.go, but I thought it doesn't make much sense / wouldn't actually help much if I add all the log flag stuff we have in main.go for this init func of a test util (as we would never set the flags)

xref: https://github.com/kubernetes-sigs/cluster-api/blob/8d4a510d3b21798829c93c753c8880e511bfe8ed/main.go#L239-L242

Basically ValidateAndApply reads the flags and then configures some logger. This logger is then used through klog.Background()

Reply:
Fine for me 👍